test/test_framework_test.go

```diff
@@ -547,7 +547,7 @@ func TestUsingEnv(t *testing.T) {
             }
         `
 
-		const scriptCode = `
+		scriptCode := `
```

Comment:
I think we can keep these as constants

Reply:
Fixed in https://github.com/onflow/cadence-tools/pull/114/commits/76657ef5bbf09b890142fd4f6dc0857098f9338a